### PR TITLE
Clarify wording about DHCP reservations

### PR DESCRIPTION
**Pull request by eiddor**
Changed "static IP" to "static IP reservation" to clarify the instruction re #204.

Signed-off-by: Roddie Hasan <roddie@krweb.net>

### CODE REVIEW

**Review by klutchell — CHANGES_REQUESTED** [review]
Could you also change the same line in `balena.yml`?

**Comment by eiddor** [comment]
Done!

File: balena.yml [hunk 1]
```diff
@@ -9,7 +9,7 @@ post-provisioning: >-
   
   When it's done you should be able to access the access the app at http://pihole.local with a default password of `balena`.
 
-  On your router or DHCP server assign a static IP to your Pi-hole device, and set your clients DNS to the same IP address.
+  On your router or DHCP server assign a static IP resreservation (if possible) to your Pi-hole device, and set your clients DNS to the same IP address.
```
Comment (klutchell) [inline balena.yml:12]:
```suggestion
  On your router or DHCP server assign a static IP reservation (if possible) to your Pi-hole device, and set your clients DNS to the same IP address.
```

**Comment by klutchell** [comment]
Could you fix the typos and squash the commits into one?

File: balena.yml (hunk 1 above)
Comment (eiddor) [reply balena.yml]: I need to not submit changes while I'm playing Rocket League.  I'll clean these up in a few after I remember how to rebase.

**Comment by klutchell** [comment]
TIP: when you squash, it combines your two commit messages into one, which is often not what you actually want (despite often using the verb squash). A better approach is `fixup` that only keeps the first commit message.

In your current state, you can just edit the last commit message with `git commit --amend` and `git push -f`

**Comment by eiddor** [comment]
One day you're going to give up on me with this stuff haha

**Comment by klutchell** [comment]
> One day you're going to give up on me with this stuff haha

never.

**Review by klutchell — APPROVED** [review]

**Pull request merged**